requirements.txt

```diff
@@ -0,0 +1,15 @@
+bcolz
+h5py
+matplotlib
+moto
```

Comment:
Do we want to include testing dependencies like `moto`?

Reply:
Debated it, but included them initially as that was easier. Wasn't obvious to me where to draw the line. Please feel free to advise.